### PR TITLE
Fix pseudo-code

### PR DESCRIPTION
**Pull request by nihaals**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,13 +81,13 @@ for b in input_stream:
                 o.append(🫂)
                 v = v - 200
             else if v >= 50:
-                o.append(🫂)
+                o.append(💖)
                 v = v - 200
```
Comment (numbermaniac) [inline README.md]:
```suggestion
                v = v - 50
```

File: README.md [hunk 2]
```diff
@@ -81,13 +81,13 @@ for b in input_stream:
...
             else if v >= 10:
-                o.append(🫂)
+                o.append(✨)
                 v = v - 200
```
Comment (numbermaniac) [inline README.md]:
```suggestion
                v = v - 10
```

File: README.md [hunk 3]
```diff
@@ -81,13 +81,13 @@ for b in input_stream:
...
             else if v >= 5:
-                o.append(🫂)
+                o.append(,)
                 v = v - 200
```
Comment (numbermaniac) [inline README.md]:
```suggestion
                v = v - 5
```

File: README.md [hunk 4]
```diff
@@ -81,13 +81,13 @@ for b in input_stream:
...
             else if v >= 1:
                 o.append(🫂)
```
Comment (numbermaniac) [inline README.md]: This one needs to be changed too; and the line below changed to `v - 1`.

File: README.md [hunk 5]
```diff
@@ -81,13 +81,13 @@ for b in input_stream:
...
-                o.append(🫂)
+                o.append(💖)
                 v = v - 200
             else if v >= 10:
-                o.append(🫂)
+                o.append(✨)
                 v = v - 200
             else if v >= 5:
-                o.append(🫂)
+                o.append(,)
```
Comment (numbermaniac) [inline README.md]: Comma is for the value 1.
5 should be the 🥺 face according to the table.

**Review by numbermaniac — CHANGES_REQUESTED** [review]

**Comment by nihaals** [comment]
Oops, I didn't look in-depth, I just assumed it was only the emoji that were wrong
Thanks @numbermaniac

**Review by numbermaniac — APPROVED** [review]

**Comment by redgoldlace** [comment]
I'll do this locally.

**Pull request closed without merging**